src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -72,6 +79,8 @@ def update_data(self):
             port_id = self.if_bpid_map[bridge_port_id]
 
             vlanmac = self.fdb_vlanmac(fdb)
```

Comment:
should we just "try" these assignments, and capture exceptions instead of making branches in fdb_vlanmac().  And if fdb_vlanmac() is not successful, the rest assignment below should be ignored too.

Reply:
Actually linters found the corner cases, and I fixed at the same time to make linting clean.
If you are worried about the CPU cycles, I tried minimize it for the normal cases.
I fixed the loop continue part.

---
In reply to: [564917045](https://github.com/Azure/sonic-snmpagent/pull/193#discussion_r564917045) [](ancestors = 564917045)